auto2.c

```diff
@@ -905,6 +909,41 @@ void auto2_read_repo_files(url_t *url)
 
 
 /*
+ * Get various files from repo-md repositrory for yast's convenience.
+ *
+ * Well, atm it's just license.tar.gz.
+ */
+void auto2_read_repomd_files(url_t *url)
+{
+  int i;
+  char *tmp_file = NULL;
+  static char *default_list[][2] = {
+    { "license", "/license.tar.gz" },
+  };
+
+  for(i = 0; i < sizeof default_list / sizeof *default_list; i++) {
+    // be careful not to replace an existing file unless we successfully got
+    // a new version
+
+    // first, get real file name
```

Comment:
if it is only comment, I suggest to remove "first" as I try to search for second which I do not see.